rancher/ecm-distro-tools
# #246: rancher image check

Adds a rancher image check command, it checks if there are docker images and helm charts for that version
Closes: #246 

## How to test
* ```
    git clone https://github.com/Tashima42/ecm-distro-tools.git
   ```
* ```
    git switch 246-rancher-image-check
   ```
* ```
    make rancher_release
    ```
* ```
    cmd/rancher_release/bin/rancher_release check-rancher-image --tag v2.7.7-rc6
    ```
